account_invoice_report_payment_info/views/report_invoice.xml

```diff
@@ -5,7 +5,9 @@
         inherit_id="account.report_invoice_document_with_payments"
     >
         <xpath expr="//t[@t-foreach='payments_vals']/tr/td" position="inside">
-            <small t-esc="payment_vals['extra_info']" />
+            <t t-if="payment_vals['extra_info']">
```

Comment:
It's really necessary to add `<t t-if `? (allways extra_info is defined according to `_get_reconciled_info_JSON_values` right?)

Reply:
Yes sorry, I set that to try to avoid what you said in last comment before investigate it, and i forget to delete it. Removed